actions/views/onboarding_tasks.ts

```diff
@@ -45,6 +47,21 @@ export function openInvitationsModal(timeout = 1) {
     };
 }
 
+export function openWorkTemplatesModal() {
```

Comment:
any particular reason for using the plural and on import changing it to singular ?

Reply:
Yeah the reason is that my IDE sucks at refactoring LOL changing it all to singular, thanks @mhd-sln